### ISSUE TITLE
FR: [Can we have split screen function on android eink device?] 

### ISSUE THREAD

**Issue by HoanPiTa** [koreader/koreader#7422]
I think it really helpful for reading bilingual language. On my boox nova 2 i am using moon reader for reading 2 books in two languages

**Comment by pazos** [comment]
Yup, see https://github.com/koreader/android-luajit-launcher/issues/238 and [a recent discussion](https://github.com/koreader/koreader/discussions/7417)

The bottom line is: no current contributor has intentions of doing the work for a final feature. Patches are very welcome

**Comment by pazos** [comment]
And that covers split mode between two different apps (one being KO). The option to run more than one instance of KO at the same time will never happen, as we're a single instance app for a good reason.

**Comment by thinh105** [comment]
> And that covers split mode between two different apps (one being KO). The option to run more than one instance of KO at the same time will never happen, as we're a single instance app for a good reason.

Just curious, can one instance of KO render 2 epubs at the same time?

**Comment by NiLuJe** [comment]
Technically, yes, but there's no UI for that.

**Comment by pazos** [comment]
The single instance thingy is a limitation of some OSes. MacOS is always single instance (albeit any app can have multiple windows and the life of the app doesn't finish when the last window is closed).

On android it is a limitation of the NativeActivity framework. The 99'999999999999999999% of apps just run a bunch of activities that can be instanciated multiple times and are destroyed and recreated on each config change.

On linux and other sane OSes you can run as many instances as you wish (I don't know if it is safe for settings et al, BTW)

**Comment by Frenzie** [comment]
It could be confusing but it should be mostly safe. Not sure about sqlite
though.
